test/unit/babel-plugin-next-ssg-transform.unit.test.js

```diff
@@ -1,507 +1,32 @@
 /* eslint-env jest */
-import { transform } from '@babel/core'
```

Comment:
Maybe we shouldn't stop testing babel until we are ready to get rid of it? I'd say just test the new transforms with Rust.

Reply:
It seems better to me.